### PR TITLE
Add endpoint for getting active games

### PR DESCRIPTION
**Pull request by jacobchrismarsh**
dependent on #128

### CODE REVIEW

File: chess/backend/src/game/logic.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from django.contrib.auth.models import User
 from django.core.handlers.wsgi import WSGIRequest
+from django.db.models import Q
```
Comment (gaganza) [inline chess/backend/src/game/logic.py]: What is Q?

File: chess/backend/src/game/logic.py [hunk 2]
```diff
@@ -207,17 +208,53 @@ def _convert_SAN_str_to_move(san: str) -> Move:
...
-    move = Moves.objects.create(**move.items())
\ No newline at end of file
+    move = Moves.objects.create(**move.items())
+
+
+def get_current_games(request: WSGIRequest) -> JsonResponse:
+    user = get_user_info(request)
+    all_games = Games.objects.filter(Q(user_id_1=user.id) | Q(user_id_2=user.id))
+
+    active_games = all_games.filter(ongoing=1)
```
Comment (gaganza) [inline chess/backend/src/game/logic.py]: maybe:

return list(map(lambda game: build_game_status_dict(game, user), all_games.filter(ongoing=1)) 

?

File: chess/backend/src/game/logic.py [hunk 3]
```diff
@@ -207,17 +208,53 @@ def _convert_SAN_str_to_move(san: str) -> Move:
...
+
+def build_game_status_dict(game: Games, user: Users) -> Dict[str, str]:
+    most_recent_move = _get_most_recent_move(game.id)
+
+    game_dict = {}
+    game_dict["id"] = game.id
+    game_dict["color"] = WHITE if game.white_user_id == user.id else BLACK
+    game_dict["turn"] = BLACK if most_recent_move.turn == WHITE else WHITE
+    game_dict["count"] = most_recent_move.move_number
+
```
Comment (gaganza) [inline chess/backend/src/game/logic.py]: maybe:

game_dict = {
   'id': game.id,
   'color': ....,
   ...
}

File: chess/backend/src/game/logic.py (hunk 1 above)
Comment (jacobchrismarsh) [reply chess/backend/src/game/logic.py]: A django way to specify queries. You need to use a `Q` object to do `OR` and `AND` statements in your query

File: chess/backend/src/game/logic.py (hunk 2 above)
Comment (jacobchrismarsh) [reply chess/backend/src/game/logic.py]: Definitely a snazzy, pythonic one-liner, but I'd argue that it sacrifices readability

File: chess/backend/src/game/logic.py (hunk 3 above)
Comment (jacobchrismarsh) [reply chess/backend/src/game/logic.py]: I thought about this, but decided against it. Looking at it again, however, I agree with you. I've updated it.

**Pull request merged**